app/pages/ServiceDiscoveryForm/ServiceDiscoveryForm.jsx

```diff
@@ -34,20 +33,44 @@ ServiceDiscoveryForm.propTypes = {
   }).isRequired,
 };
 
-
 /** Main component that handles form data and advancing steps. */
-const InnerServiceDiscoveryForm = ({
-  steps, categorySlug, eligibilities, subcategories, subcategorySubheading,
-}) => {
+function InnerServiceDiscoveryForm({
+  steps, eligibilities, subcategories, categorySlug, subcategorySubheading,
+}) {
   const [currentStep, setCurrentStep] = useState(0);
-  const [selectedEligibilities, setSelectedEligibilities] = useState({});
-  const [selectedSubcategories, setSelectedSubcategories] = useState({});
 
   // TODO: Should goBack go back to the previous step?
   const history = useHistory();
   const goBack = () => history.goBack();
   const goToNextStep = () => setCurrentStep(currentStep + 1);
 
+  return (
+    <>
+      <Header onGoBack={goBack} />
+      <Content
+        steps={steps}
+        currentStep={currentStep}
+        eligibilities={eligibilities}
+        subcategories={subcategories}
+        categorySlug={categorySlug}
+        subcategorySubheading={subcategorySubheading}
+      />
+      <Footer
+        onGoBack={goBack}
+        onNextStep={goToNextStep}
+        currentStep={currentStep}
+        numSteps={steps.length}
+      />
+    </>
+  );
+}
+
+function Content({
```

Comment:
This was a violation of the `no-unstable-nested-components` rule that seemed like a good idea to go ahead and quickly refactor. I extracted the `Content` component and moved things into lower scope that seemed unneeded in higher scope. It required a little bit of prop drilling.